services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/entity/MacRange.java

```diff
@@ -14,12 +14,17 @@
 */
 package com.futurewei.alcor.macmanager.entity;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.futurewei.alcor.macmanager.utils.MacUtil;
 import lombok.Data;
 
+import java.util.BitSet;
+
 
 @Data
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.NONE,
```

Comment:
Just out of curiosity: what is @JsonAutoDetect for?

Reply:
It is to prevent duplicated (POJO to JSON) convert. 
Referenece: https://github.com/FasterXML/jackson-databind/issues/1609